doc/api/modules.md

```diff
@@ -888,12 +888,29 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.createRequireFromPath(filename)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string} Filename to be used to construct the relative require
+  function.
+* Returns: {[`require`][]} Require function
+
+```js
+const { createRequireFromPath } = require('module');
+const requireUtil = createRequireFromPath('../src/utils');
+
+requireUtil('zalgo');
+```
```

Comment:
👆 Could we use another name than `zalgo`
> Zalgo is an Internet legend about an ominous entity believed to cause insanity, death and destruction of the world, similar to the creature Cthulhu 

That's seems awful heavy for this util method 🙃

Reply:
i didn't create the file, its just part of our fixtures.

Reply:
was a reference to https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags but i can change it if you really want 🙃

Reply:
Ye̾s̈́̀̈́̎̾̈́ ͂p̏̒ͬlͬeͫ͌̊͋ả̆̄̌͆sẽ̚

Reply:
aww, RIP whimsy